### ISSUE TITLE
Elib

### ISSUE THREAD

**Issue by kevinvenson** [kevinvenson/elib-Dapp#4]
<!DOCTYPE html>
<html lang="en">
<head>
 <title>Bootstrap Example</title>
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 <style>
   /* Remove the navbar's default margin-bottom and rounded borders */
   .navbar {
     margin-bottom: 0;
     border-radius: 0;
   }

   /* Set height of the grid so .sidenav can be 100% (adjust as needed) */
   .row.content {height: 450px}

   /* Set gray background color and 100% height */
   .sidenav {
     padding-top: 20px;
     background-color: #f1f1f1;
     height: 100%;
   }

   /* Set black background color, white text and some padding */
   footer {
     background-color: #555;
     color: white;
     padding: 15px;
   }

   /* On small screens, set height to 'auto' for sidenav and grid */
   @media screen and (max-width: 767px) {
     .sidenav {
       height: auto;
       padding: 15px;
     }
     .row.content {height:auto;}
   }
 </style>
</head>
<body>

<nav class="navbar navbar-inverse">
 <div class="container-fluid">
   <div class="navbar-header">
     <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
       <span class="icon-bar"></span>
       <span class="icon-bar"></span>
       <span class="icon-bar"></span>
     </button>
     <a class="navbar-brand" href="#">eLibrary</a>
   </div>
   <div class="collapse navbar-collapse" id="myNavbar">
     <ul class="nav navbar-nav">
       <li class="active"><a href="#">Home</a></li>
       <li><a href="#">Publishers</a></li>
       <li><a href="#">Verifiers</a></li>
       <li><a href="#">Books</a></li>
     </ul>
     <ul class="nav navbar-nav navbar-right">
       <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
     </ul>
   </div>
 </div>
</nav>

<div class="container-fluid text-center">
 <div class="row content">
   <div class="col-sm-8 text-left">
     <h1>eLibrary</h1>
     <p style="font-size:150%;">Register as Publisher</p>
        Address:<br>
        <input type="text" class="form-control" name="Address"><br>
        Publisher's Name:<br>
        <input type="text" class="form-control" name="Publisher's Name">
        <button type="submit" class="btn btn-primary mb-3" v-on:click="Submit">Submit</button>
        <button class="btn btn-primary mb-3">Clear</button>
        </form>
     <hr>
     <p style="font-size:150%;">Register as Verifier</p>
     <form>
        Address:<br>
        <input type="text" class="form-control" name="Address"><br>
        Verifier's Name:<br>
        <input type="text" class="form-control" name="Verifier's Name">
        <button type="submit" class="btn btn-primary mb-3" v-on:click="Submit">Submit</button>
        <button class="btn btn-primary mb-3">Clear</button>
     </form>
     <hr>
     <p style="font-size:150%;"> Add a Book</p>
     <form>
        Publisher's Address:<br>
        <input type="text" class="form-control" name="Address"><br>
        Title of the book:<br>
        <input type="text" class="form-control" name="Book's Name"><br>
        Author's Name:<br>
        <input type="text" class="form-control" name="Author's Name">
        <button type="submit" class="btn btn-primary mb-3" v-on:click="Submit">Submit</button>
        <button class="btn btn-primary mb-3">Clear</button>
     </form>
     <hr>

     <p>Lorem ipsum...</p>
   </div>
 </div>
</div>

<footer class="container-fluid text-center">
 <p>Brought to you by: Lorem ipsum</p>
</footer>

</body>
</html>
